x/staking/spec/delegation_sequence.puml

```diff
@@ -0,0 +1,94 @@
+@startuml
```

Comment:
Can we put this in a subdirectory ?

Reply:
Preferably in a subdirectory in `docs/`.